### PR TITLE
'inspect' command: display master apps of a given node host

### PR DESCRIPTION
**Pull request by pyseed**

### CODE REVIEW

**Comment by petrus-v** [comment]
Thanks for this works!

Do you mind to add a test case to [improve coverall](https://coveralls.io/builds/20260355/source?filename=cluster/cluster.py#L268) and avoid regression on this new features ?

I guess you can find inspiration how to do it from [this check test](https://github.com/mlfmonde/cluster_cli/blob/master/cluster/tests/test_checks.py#L119) test to assert consol output.

And you could probably reuse/refactor [fill_data method](https://github.com/mlfmonde/cluster_cli/blob/master/cluster/tests/test_checks.py#L15).

Next evolution could be to add a --format option to let user choose wich fields to add per service, for instance it could be nice to display the deploy date or deploy id. But probably doing that in an other PR, what do you think about ?

Regards

**Comment by pyseed** [comment]
Thanks for input.

Yes i will fix coverage in same PR, fields improvement will go in another PR.

**Comment by pyseed** [comment]
@petrus-v Hi. coverage improved, PR candidate for a merge.

**Comment by pyseed** [comment]
@petrus-v readme/changelog updated, PR candidate for merge

File: README.md [hunk 1]
```diff
@@ -42,6 +43,7 @@ positional arguments:
                         host server.This command will helps you to send all
```
Comment (petrus-v) [inline README.md]: few lines above **inspect** is missing in the list ``{checks,deploy,migrate,move-masters-from}`` (this list appear twice)

Comment (pyseed) [reply README.md]: done !

**Comment by petrus-v** [comment]
LGTM, thanks for that feature

**Pull request merged**